support/mender-inventory-update-modules

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Returns an the update-modules currently installed on the system
+# For more information on update-modules see:
+# https://hub.mender.io/c/update-modules/13
+#
+
+set -e
+
+FOUND=0
+
+if [ ! -d /usr/share/mender/modules/ ]; then
+  echo update_module'(s)'=""
```

Comment:
Aren't we just asking for shell parsing trouble here? I'm not against it per se, but I just reckon `update_modules` is just as clear and correct, less likely to be misunderstood as some shell magic, and easier to handle if you ever want to query the inventory from a script.

Reply:
Agreed!